README.md

```diff
@@ -122,27 +123,52 @@ to bind archetypes.
 
 ... and from here you are set for `builder`-controlled meta goodness!
 
-#### Builder Commands
+#### Builder Actions
 
-Display general or command-specific help.
+Display general or command-specific help (which shows available specific flags).
 
 ```sh
 $ builder help
-$ builder help run
+$ builder help ACTION
 ```
 
-Run a single `package.json` `scripts` task.
+Run `builder help ACTION` for all available options. For a quick overview:
+
+##### builder run
+
+Run a single task from `script`. Analogous to `npm run TASK`
 
 ```sh
-$ builder run foo-task
+$ builder run TASK
```

Comment:
@chaseadamsio :+1: Both `git` and `npm` follow this convention, as minor as it is, I'd lean towards copying them. `builder run <task>`